server/templates/base.html

```diff
@@ -39,9 +39,10 @@
   <script async src="https://www.googletagmanager.com/gtag/js?id={{ GA_ACCOUNT }}"></script>
   <script>
     window.dataLayer = window.dataLayer || [];
-    function gtag(){dataLayer.push(arguments);}
+    function gtag() { dataLayer.push(arguments); }
     gtag('js', new Date());
-    gtag('config', '{{ GA_ACCOUNT }}', { client_storage: 'none', anonymize_ip: true });
+    gtag('config', '{{ GA_ACCOUNT }}', { client_storage: 'none', anonymize_ip: true, place_category: new URLSearchParams(window.location.search).get('category') || '' });
```

Comment:
please use category that's already parsed in flask. you can pass in new arguments to the template (and give it a specific name like `place_category` since this will be used across all templates).

Reply:
Done